enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -23,7 +23,7 @@ export const PACKAGE_JSON_NOT_FOUND_MESSAGE = `
 
 export const MISSING_REACT_DEPENDENCY = `
   Your package.json file does not contain a dependency for React.
-  Please make sure your package.json file contains a dependency for React 18.
+  Please make sure your package.json file contains a dependency for React 18 - 19.
```

Comment:
should we use the same phrase from the first file?
```suggestion
  Please make sure your package.json file contains a dependency for React 18 or React 19.
```

Reply:
Maybe this is justified because we're focusing on the version number on package.json.

Reply:
Fixed, thanks!